src/cmd/top/top.c

```diff
@@ -385,6 +385,9 @@ int main (int argc, char *argv[])
                             optparse_get_str (opts, "queue", NULL),
                             &error)))
         fatal (0, "%s", error.text);
+    /* Setup test-exit after top_create() call above, we do not want
+     * the very first "draw" to result in any testing output.
+     */
```

Comment:
This is still slightly confusing. What is the testing output -- test-exit just tells `flux-top(1)` to exit after all panes have been drawn once? Actually now that test-exit ensures each pane has been drawn once, _could_ it be set before `top_create()`? (I assume you tried this and thus the comment, so I'm probably still missing some key information - I didn't look at the code in detail)

Reply:
I agree the comment will be helpful for future programmers, I was just still a bit confused by the comment (since I didn't look at the code) so wondered if it needed a slight tweak? Maybe just expanding the explanation (and include the use case for --test-exit, i.e. why it is used) would be enough.